Sources/UnleashProxyClientSwift/Poller/PollerError.swift

```diff
@@ -1,7 +1,7 @@
 public enum PollerError: Error {
-    case decoding
-    case network
+    case decoding(Error)
```

Comment:
since we're changing the shape of the error API here it'd probably require a major version bump of the library, correct?

Reply:
@FredrikOseberg FYI